robottelo/cli/base.py

```diff
@@ -374,6 +374,13 @@ def _construct_command(cls, options=None):
                 if val is True:
                     tail += u' --{0}'.format(key)
                 elif val is not False:
+                    if isinstance(val, list):
+                        val = ','.join(
+                            str(el.get('id') or el.get('name'))
```

Comment:
We can join on this method but making it decide to get id or name is something that could cause doubt. Is better to leave this on the test so we know that we are passing a list of ids or a list of names.